code/modules/mob/living/living.dm

```diff
@@ -957,8 +956,8 @@
 	else if(has_legs || ignore_legs)
 		lying = 0
 
-	if(buckled)
-		lying = 90*buckle_lying
+	if(buckled && buckled.buckle_lying != -1)
```

Comment:
I believe it should be 0 here. -1 is buckle to the left, 1 is buckle to the right.

Reply:
I see. The comment on the var led me to believe -1 should let you stand or lie while buckled.